### ISSUE TITLE
One missing configuration setting

### ISSUE THREAD

**Issue by Shengcao-Cao** [ChenhongyiYang/CCOP#2]
Thanks for your great work and releasing the code!

During reproducing the experimental results, I noticed a tiny discrepancy between your code and the standard Mask R-CNN fine-tuning:

In your configuration file `benchmarks/detection/configs/coco_R_50_FPN_1x_infomin.yaml`, the configuration `MODEL.ROI_BOX_HEAD.POOLER_RESOLUTION` is not set. As a result, Detectron2 will use the default setting `MODEL.ROI_BOX_HEAD.POOLER_RESOLUTION = 14`, from https://github.com/facebookresearch/detectron2/blob/main/detectron2/config/defaults.py. However, this does not match the standard Mask R-CNN setting `MODEL.ROI_BOX_HEAD.POOLER_RESOLUTION = 7`. Please see https://github.com/facebookresearch/detectron2/blob/main/configs/Base-RCNN-FPN.yaml.

I'm not sure which Detectron2 version you were previously using, so this difference could be a result of Detectron2 updates as well.

**Comment by ChenhongyiYang** [comment]
Hi. Thank you for trying our code :-) 

In our paper, we set MODEL.ROI_BOX_HEAD.POOLER_RESOLUTION to 7 and MODEL.ROI_MASK_HEAD.POOLER_RESOLUTION to 14, which is the standard implementation. If you set MODEL.ROI_BOX_HEAD.POOLER_RESOLUTION to 14, you will get a higher AP performance. 

I have just checked the default setting of Detectron2 and find it is indeed 14, however, I clearly remember that the original default was 7. I do not know when did they change the setting. 

Anyway, we can make sure that we did set the MODEL.ROI_BOX_HEAD.POOLER_RESOLUTION to 7 during our experiments and all the numbers in the paper were based on this setting.

**Comment by Shengcao-Cao** [comment]
Thanks for your reply!

Detetron2 sets `MODEL.ROI_BOX_HEAD.POOLER_RESOLUTION = 14` by default, which is intended for C4-based detectors rather than FPN (probably with some historical reasons I guess: https://github.com/facebookresearch/Detectron/issues/118). Anyway, we can include this line into the configuration file. You may check this configuration in previous experiments if you still keep the original training log.

Best regards.
